evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -248,7 +249,7 @@ impl<'a, B: Backend> CheatcodeStackExecutor<'a, B> {
                 if !self.enable_ffi {
                     return evm_error(
                         "ffi disabled: run again with --ffi if you want to allow tests to call external scripts",
-                    )
+                    );
```

Comment:
Not needed - if you can, avoid drive by formatting changes like this which are unrelated to the PR

Reply:
I use vim with coc and rust-analyzer and formats automatically the whole source file. 

e.g if I add a package, it will automatically place it in the correct alphabetical order. Need to look into it.